pages/myforms/[[...path]].tsx

```diff
@@ -71,51 +72,56 @@ const RenderMyForms: NextPageWithLayout<MyFormsProps> = ({ templates }: MyFormsP
   }, []);
 
   return (
-    <div id="page-container" className="lg:!mx-4 xl:!mx-8">
-      <div>
-        <LeftNavigation />
-        <main id="content" className="ml-60 xl:ml-40 md:pl-5">
-          <h1 className="border-b-0 mb-8 md:text-h1">{t("title")}</h1>
-          <div className="top-40">
-            <ResumeEditingForm>
-              <StyledLink href="/form-builder/edit" className="mr-8">
-                <span aria-hidden="true"> ← </span> {t("actions.resumeForm")}
-              </StyledLink>
-            </ResumeEditingForm>
-            <div ref={createNewFormRef} className="inline">
-              <StyledLink href="/form-builder">
-                <span aria-hidden="true">+</span> {t("actions.createNewForm")}
-              </StyledLink>
+    <>
+      <Head>
+        <title>{t("title")}</title>
+      </Head>
+      <div id="page-container" className="lg:!mx-4 xl:!mx-8">
+        <div>
+          <LeftNavigation />
+          <main id="content" className="ml-60 xl:ml-40 md:pl-5">
+            <h1 className="border-b-0 mb-8 md:text-h1">{t("title")}</h1>
+            <div className="top-40">
+              <ResumeEditingForm>
+                <StyledLink href="/form-builder/edit" className="mr-8">
+                  <span aria-hidden="true"> ← </span> {t("actions.resumeForm")}
+                </StyledLink>
+              </ResumeEditingForm>
+              <div ref={createNewFormRef} className="inline">
+                <StyledLink href="/form-builder">
+                  <span aria-hidden="true">+</span> {t("actions.createNewForm")}
+                </StyledLink>
+              </div>
             </div>
-          </div>
-          <TabPanel id="tabpanel-drafts" labeledbyId="tab-drafts" isActive={path === "drafts"}>
-            {templatesDrafts && templatesDrafts?.length > 0 ? (
-              <CardGrid cards={templatesDrafts}></CardGrid>
-            ) : (
-              <p>{t("cards.noDraftForms")}</p>
-            )}
-          </TabPanel>
-          <TabPanel
-            id="tabpanel-published"
-            labeledbyId="tab-published"
-            isActive={path === "published"}
-          >
-            {templatesPublished && templatesPublished?.length > 0 ? (
-              <CardGrid cards={templatesPublished}></CardGrid>
-            ) : (
-              <p>{t("cards.noPublishedForms")}</p>
-            )}
-          </TabPanel>
-          <TabPanel id="tabpanel-all" labeledbyId="tab-all" isActive={path === "all"}>
-            {templatesAll && templatesAll?.length > 0 ? (
-              <CardGrid cards={templatesAll}></CardGrid>
-            ) : (
-              <p>{t("cards.noForms")}</p>
-            )}
-          </TabPanel>
-        </main>
+            <TabPanel id="tabpanel-drafts" labeledbyId="tab-drafts" isActive={path === "drafts"}>
```

Comment:
Something is weird with the following block of code. Looks like it was indented using spaces instead of tabs